clarity_ext/context.py

```diff
@@ -9,9 +7,10 @@
 from clarity_ext.repository import StepRepository
 from clarity_ext import utils
 from clarity_ext.driverfile import OSService
+from clarity_ext.domain.udf import Udf
 
 
-class ExtensionContext(object):
+class ExtensionContext(Udf):
```

Comment:
... but as I mentioned in the other PR, I have some pending changes for UDFs, so I don't see any need for renaming this.